.github/Pull_Request_template.md

```diff
@@ -10,6 +10,12 @@ _Benchmarks PR (if needed): https://github.com/IntelPython/scikit-learn_bench/pu
 
 ---
 
+PR should start as a draft, then move to ready for review state after CI passed and checkboxes closed.
+This approach ensures that reviewers don't spend extra time requesting changes for regular appearing things.
```

Comment:
What does "regular appearing things" mean here?

Reply:
- "Add comment explaining this hard-to-understand part"
- "Provide perf. measurements"
- "Update documentation to reflect changes"
- "Add test for this function"